frontend/src/v5/ui/routes/dashboard/projects/federations/editFederationModal/editFederationModal.component.tsx

```diff
@@ -58,10 +51,13 @@ export const EditFederationModal = ({
 		event.preventDefault();
 		onClickClose();
 	};
-
+	const formIsDirty = () => !isEqual(
+		compact(includedContainers).map((c) => c._id),
```

Comment:
The `compact` method is used to filter out falsey values, but do we ever get them here?

Reply:
You're right. You used to briefly get undefined values when the form was loading but I fixed it closer to the source of the problem by adding the `if (containers.length)` in editFederation.component. I forgot to remove this "compact" solution, but have now fixed it